src/palm-react/hooks/lens/useLens.ts

```diff
@@ -343,8 +343,8 @@ const useLens = (): UseLensReturn => {
               s,
               deadline: typedData.value.deadline,
             },
-          },
-          { gasLimit: 8000000, gasPrice: utils.parseUnits('90', 'gwei') }
+          }
+          // { gasLimit: 8000000, gasPrice: utils.parseUnits('90', 'gwei') }
```

Comment:
?

Reply:
This is the deletion of the gas limit of tx when performing update profile nft. It's temporary until I add estimation gas that all tx.